src/index.html

```diff
@@ -5,6 +5,7 @@
     <title>OpenStreetMap Americana</title>
     <link rel="preconnect" href="https://osm-americana.github.io" />
     <link href="fonts.css" rel="stylesheet" />
+    <script type="module" src="js/load_fonts.js"></script>
```

Comment:
Do we need to document this additional script as an installation step somewhere?